src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControl.cs

```diff
@@ -0,0 +1,416 @@
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or if there happens to be an error while retrieving info of the track being played on the target.
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        public Track GetTrackInfo()
+        {
+            return BluetoothAvrcpControlImpl.Instance.GetTrackInfo();
+        }
+
+        /// <summary>
+        /// Sends a particular play command to the target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected.
+        /// </remarks>
+        /// <param name="command">Command to be sent</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when sending command to the target device fails.
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void SendPlayerCommand(PlayerCommand command)
+        {
+            BluetoothAvrcpControlImpl.Instance.SendPlayerCommand(command);
+        }
+
+        /// <summary>
+        /// Sends a play command to a particular target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected.
+        /// </remarks>
+        /// <param name="command">Command to be sent</param>
+        /// <param name="remoteAddress">Address of the device to send command</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when sending command to the target device fails.
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SendPlayerCommandTo(PlayerCommand command, string remoteAddress)
+        {
+            BluetoothAvrcpControlImpl.Instance.SendPlayerCommandTo(command, remoteAddress);
+        }
+
+        /// <summary>
+        /// Sets absolute volume of target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected
+        /// </remarks>
+        /// <param name="volume">The volume level to be set</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when setting absolute volume of the target device fails
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetAbsoluteVolume(uint volume)
+        {
+            Interop.Bluetooth.SetAbsoluteVolume(volume);
```

Comment:
Why do you directly call native API? you should handle error and report as exception
```suggestion
            BluetoothAvrcpControlImpl.Instance.SetAbsoluteVolume(volume);
```

Reply:
Done.